agent-csa-bundle/build.gradle.kts

```diff
@@ -0,0 +1,82 @@
+
+plugins {
+  id("maven-publish")
+  id("signing")
+}
+
+dependencies {
+  implementation(project(":agent"))
+}
+
+// This should be updated for every CSA release
+val csaVersion = "25.3.0-1321"
+base.archivesName.set("splunk-otel-javaagent-csa")
+
+tasks {
+  val bundleSecureApp by registering(Exec::class) {
+    description = "Bundle the Cisco SecureApp with splunk-otel-java"
+    dependsOn(":agent:assemble")
+    isIgnoreExitValue = false
+    commandLine("./build-agent-csa-fat-jar.sh", version, csaVersion)
```

Comment:
invoking an shell script is an unusual choice, did you consider doing whatever the shell script does with gradle?

Reply:
Thanks, yeah, I concede that this isn't the greatest. We do resort to scripts in a number of other places, but I think we could probably figure this out as pure gradle. Will take that as a follow-up item.